expose

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env php
+#!/usr/bin/env /usr/sbin/php
```

Comment:
Why did you add this change?

Reply:
I thought it was a merge with the master branch, but actually I have no idea where this was coming from. I reverted that commit.